website/static/js/clipboard.js

```diff
@@ -1,26 +1,33 @@
 'use strict';
 
 var $ = require('jquery');
-var ZeroClipboard = require('zeroclipboard');
+var Clipboard = require('clipboard');
 
-ZeroClipboard.config({
-    swfPath: '/static/vendor/bower_components/zeroclipboard/dist/ZeroClipboard.swf'
-});
+var setTooltip = function (elm, message) {
+    $(elm).tooltip('hide')
+    .attr('title', message)
+    .tooltip('show');
+};
+
+var hideTooltip = function (elm) {
+    setTimeout(function() {
+        $(elm).tooltip('hide');
+    }, 2000);
+};
 
 var makeClient = function(elm) {
     var $elm = $(elm);
-    var client = new ZeroClipboard(elm);
 
-    $elm.on('mouseover', function() {
-        $elm.addClass('active');
-    });
-    $elm.on('mouseout', function() {
-        $elm.removeClass('active');
+    var client = new Clipboard(elm);
+
+    client.on('success', function(e){
+        setTooltip(e.trigger, 'Copied!');
```

Comment:
Since you refactored the tooltips here, you should mention in your changes so QA will be sure to test it.